src/amo/components/AddonMeta.js

```diff
@@ -49,7 +55,14 @@ export class AddonMetaBase extends React.Component {
         <div className="AddonMeta-item AddonMeta-users">
           <h3 className="visually-hidden">{i18n.gettext('Used by')}</h3>
           <p className="AddonMeta-text AddonMeta-user-count">
-            {userCount}
+            {addon && isAddonAuthor({ addon, userId }) ? (
```

Comment:
There is probably no need for `addon &&` because it is checked in the `isAddonAuthor()` function.

Reply:
Ah, true!

Reply:
Annoyingly, flow can't understand I won't access that property unless `isAddonAuthor` is `true` (which it never would be if `addon` were `null`), so I left this in.